app/assets/javascripts/discourse/tests/acceptance/group-index-test.js

```diff
@@ -83,7 +83,51 @@ acceptance("Group Members", function (needs) {
 
     const memberDropdown = selectKit(".bulk-group-member-dropdown");
     await memberDropdown.expand();
-    await memberDropdown.selectRowByValue("makeOwners");
+
+    assert.ok(
+      exists('[data-value="removeMembers"]'),
+      "it includes remove member option"
+    );
+
+    assert.ok(
+      exists('[data-value="makeOwners"]'),
+      "it includes make owners option"
+    );
+
+    assert.ok(
+      exists('[data-value="setPrimary"]'),
+      "it includes set primary option"
+    );
+  });
+
+  test("Shows bulk actions as a group owner", async function (assert) {
+    updateCurrentUser({ moderator: false, admin: false });
+
+    await visit("/g/discourse");
+
+    assert.ok(exists("button.bulk-select"));
```

Comment:
Nitpick: I believe this is not necessary because if it does not exist, the next line will fail anyway.

Reply:
Good point! 😅